rmgpy/statmech/rotation.pyx

```diff
@@ -83,6 +84,12 @@ cdef class Rotation(Mode):
         """
         return (Rotation, (self.symmetry, self.quantum))
 
+    def make_object(self, data, class_dict):
```

Comment:
Add in rotation.pxd as well, or is the definition already there?

(Same for torsion)

Reply:
Good question, a thought the same when I first did this. Because `Rotation` inherits from `RMGObject`, this method is already present in this class (and was declared in rmgobject.pxd) and does not need to be declared in the header. All we are doing here is overwriting the parent method.

I am pretty sure I tried adding this to the header file initially, but I got an error. I'll check this one more time though.